### ISSUE TITLE
Bleeding is obscenely strong

### ISSUE THREAD

**Issue by metalgearsloth** [space-wizards/space-station-14#17021]
You can win a fight and if you're bleeding (which as of like a month or two ago will happen every time) you will almost always die before you get medical attention.

I VVd someone ingame and it was 0.2 dps which is 8 minutes from 0 damage but often it will be much faster than that. E.g. if you're at 70 damage then you only have 2.5 minutes to fix it.

**Comment by deltanedas** [comment]
yeah this basically means to not die you need to immediately weld yourself while or after fighting

**Comment by Skyedra** [comment]
Maybe it should also be easier to find gauze in the station?  Anecdotally, it feels easier to find defibs than gauze in hallways and whatnot.  Having lots of basic gauze stored in strategically located cabinets (common hallways) would buy people time to make the hike to med and get damage treated.

Also, I often don't even realize I'm taking bloodloss damage, especially given how much blood is already on the floor in an average round.  Maybe there should be a UI indicator for it, similar to airloss / temperature...

**Comment by metalgearsloth** [comment]
Bleeding probably needs its own damage overlay though brute already takes red so not sure, but it still needs a heavy nerf.

**Comment by Aisu9** [comment]
you can also use the lights in hallways to stop all bleeding, each point of heat damage remove 4 bleedrate, you can cut jumpsuit and use cloth as improvised gauzes (they stop 2.5 instead of 10 bleedrate)

**Comment by Aisu9** [comment]
more ways to heal bloodloss damage/recover faster (same as airloss?). missing a lot of blood (we are talking about slightly more than 10% of what is in your body) is extremely dangerous for us, in-game you can shrug it off (if you are not bleeding much more).
Increase maximum blood level may also be an idea, but this will make blood pack and iron pills horribly weaker

**Comment by Aisu9** [comment]
> Bleeding probably needs its own damage overlay though brute already takes red so not sure, but it still needs a heavy nerf.

Bleeding already have its own overlay, your light radium reduce as your bloodloss damage increases (pretty sure Brute damage overlay take priority and completely make BL overlay useless)

**Comment by metalgearsloth** [comment]
> > Bleeding probably needs its own damage overlay though brute already takes red so not sure, but it still needs a heavy nerf.
> 
> Bleeding already have its own overlay, your light radium reduce as your bloodloss damage increases (pretty sure Brute damage overlay take priority and completely make BL overlay useless)

A lot of stuff adjust light range so it's not really indicative of anything (asphyxiation / blindness also do).

**Comment by Aisu9** [comment]
blindness light range modification is almost non existent. you only start seeing the border of your screen getting darker and next wielding WILL make you blind without protection. (magboots of blinding speed make you straight up blind, no eye damage overtime)

**Comment by SlamBamActionman** [comment]
Closing this issue due to bleeding being less strong these days + upcoming Medical changes will modify the impact and treatment of bleeding regardless.
